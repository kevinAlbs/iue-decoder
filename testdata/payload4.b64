BG0BAAAFZAAgAAAAAIv6VUIIdmtHdpgZoJ3jI1ml1C2PRpbVhDZ+KLANYotkBXMAIAAAAAAYeqduU82pqo68MSIuOE8SLoa31j+2KEgNUJmOmMevGQVjACAAAAAAMMRH1V+7hgaihhDEhR70GldDZgQ8CDNasgLlI3T2EPUFcABQAAAAAI01AUQR/WmNQM9F0bCmYSUnlfWt/0VMG0TraZFQBTpVJ4WjJlaDS2hrCIVnlugzmeOisFFkx4ZvLGx5TXrwbJ0dRT011j1BxQHERAugUqeOBXUAEAAAAAQi4yMnD+xGdb8i/okx28IyEHQAAgAAAAV2AEkAAAAAIuMjJw/sRnW/Iv6JMdvCMnuKUnAoATZAj+50R9EyQIlO9G2HAEjOvfPb6g7OWYgFs1ogqzjXxeZDlSN1GfjQW3+nBMA6+KlrYgVlACAAAAAAOTxFECgSgD/PKSpuTxZx1vWmUY/E0XeZdG6Mk3Mmj9QA