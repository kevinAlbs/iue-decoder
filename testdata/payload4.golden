BlobSubtype=4 (FLE2InsertUpdatePayload)
EDCDerivedFromDataTokenAndCounter=8BFA554208766B47769819A09DE32359A5D42D8F4696D584367E28B00D628B64 ("d":{"$binary":{"base64":"i/pVQgh2a0d2mBmgneMjWaXULY9GltWENn4osA1ii2Q=","subType":"00"}})
ESCDerivedFromDataTokenAndCounter=187AA76E53CDA9AA8EBC31222E384F122E86B7D63FB628480D50998E98C7AF19 ("s":{"$binary":{"base64":"GHqnblPNqaqOvDEiLjhPEi6Gt9Y/tihIDVCZjpjHrxk=","subType":"00"}})
ECCDerivedFromDataTokenAndCounter=30C447D55FBB8606A28610C4851EF41A574366043C08335AB202E52374F610F5 ("c":{"$binary":{"base64":"MMRH1V+7hgaihhDEhR70GldDZgQ8CDNasgLlI3T2EPU=","subType":"00"}})
Encrypted tokens=8D35014411FD698D40CF45D1B0A661252795F5ADFF454C1B44EB699150053A552785A32656834B686B08856796E83399E3A2B05164C7866F2C6C794D7AF06C9D1D453D35D63D41C501C4440BA052A78E ("p":{"$binary":{"base64":"jTUBRBH9aY1Az0XRsKZhJSeV9a3/RUwbROtpkVAFOlUnhaMmVoNLaGsIhWeW6DOZ46KwUWTHhm8sbHlNevBsnR1FPTXWPUHFAcREC6BSp44=","subType":"00"}})
IndexKeyId=22E323270FEC4675BF22FE8931DBC232 ("u":{"$binary":{"base64":"IuMjJw/sRnW/Iv6JMdvCMg==","subType":"04"}})
Encrypted Type=2 ("t":2)
Value={"$binary":{"base64":"IuMjJw/sRnW/Iv6JMdvCMnuKUnAoATZAj+50R9EyQIlO9G2HAEjOvfPb6g7OWYgFs1ogqzjXxeZDlSN1GfjQW3+nBMA6+KlrYg==","subType":"00"}} ("v":{"$binary":{"base64":"IuMjJw/sRnW/Iv6JMdvCMnuKUnAoATZAj+50R9EyQIlO9G2HAEjOvfPb6g7OWYgFs1ogqzjXxeZDlSN1GfjQW3+nBMA6+KlrYg==","subType":"00"}})
ServerDataEncryptionLevel1Token=393C45102812803FCF292A6E4F1671D6F5A6518FC4D17799746E8C9373268FD4 ("e":{"$binary":{"base64":"OTxFECgSgD/PKSpuTxZx1vWmUY/E0XeZdG6Mk3Mmj9Q=","subType":"00"}})
